README.md

```diff
@@ -182,8 +182,8 @@ could a mismatch between provided SPID and api key as specified below).
 
 Place your ias api key and your SPID in the ``ias`` folder as follows:
 
-    echo 'YOUR_API_KEY' > ${GOPATH}/src/github.com/hyperledger-labs/fabric-private-chaincode/fabric/sgxconfig/ias/api_key.txt
-    echo 'YOURSPID' | xxd -r -p > ${GOPATH}/src/github.com/hyperledger-labs/fabric-private-chaincode/fabric/sgxconfig/ias/spid.txt
+    echo 'YOUR_API_KEY' > ${GOPATH}/src/github.com/hyperledger-labs/fabric-private-chaincode/config/ias/api_key.txt
+    echo 'YOURSPID' | xxd -r -p > ${GOPATH}/src/github.com/hyperledger-labs/fabric-private-chaincode/config/ias/spid.txt
```

Comment:
is there a particular reason why this is now not in fabric/sgxconfig anymore? I guess this could also go into integration folder?!

Reply:
The files are actually in FPC_TOP_DIR/config/ias so there is a single place where user-configurable stuff is.  The various configs then can reference the common place (currently done via sym-links but could also be achieved by modifying the core.yaml and directly reference that dir/these files ...

Reply:
+1